thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/DetectionOnboardService.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.thirdeye.anomaly.onboard;
+
+import java.util.Map;
+
+public interface DetectionOnboardService {
```

Comment:
Will we have multiple detection onboarding services? What is the plan here? Or it is to extend it to the other job services?

Reply:
I am hoping this class will be extended to an interface for generic job executor.

Reply:
I wonder if this duplicates existing logic - i.e. could any job submission simply return the id assigned by the database, without keeping track of it in memory? What if there are multiple different frontend servers?

Reply:
This executor service does not use DB and hence the job id is assigned and tracked in its memory. The next step is to use TE's DB for distributing the tasks. At that time, the Job ID will be the one returned from DB. In addition, we assume that there is only one machine runs detection onboarding jobs.